### PR TITLE
Replace inaccessible disabled option of NcCheckboxRadioSwitch with NcNoteCard

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
Replace inaccessible disabled option of NcCheckboxRadioSwitch with NcNoteCard

* For https://github.com/nextcloud/server/issues/42941

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
image | Screenshot from 2024-01-22 11-33-24

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/settings/src/components/AdminAI.vue [hunk 1]
```diff
@@ -31,9 +31,9 @@
 				</NcCheckboxRadioSwitch>
 			</template>
 			<template v-if="!hasStt">
-				<NcCheckboxRadioSwitch disabled type="radio">
+				<NcNoteCard type="warning">
```
Comment (szaimen) [inline apps/settings/src/components/AdminAI.vue:34]: I would rather use info here and below :)

**Review by ShGKme — APPROVED** [review]

**Review by szaimen — APPROVED** [review]
LGTM :)

**Review by emoral435 — APPROVED** [review]

**Pull request merged**